File: ts-express-graphql-knex-bookshelf/models/jutsu.ts
import bookshelf from "../utils/bookshelf";

const Jutsu = bookshelf.model("Jutsu", {
  tableName: "jutsus",
  ninjas() {
    return this.belongsToMany("Ninja", "ninjas_jutsus", "jutsu_id", "ninja_id");
  },
});

export default Jutsu;

File: python-fastapi-ariadne-orator/models/jutsu.py
# connected Model
from connections.pg import Model
# orator
from orator.orm import belongs_to_many
# mixin
from orator import SoftDeletes


class Jutsu(SoftDeletes, Model):
    __table__ = 'jutsus'  # without this, orator looks for plural snake case.

    @belongs_to_many('ninjas_jutsus', 'jutsu_id', 'ninja_id')
    def ninjas(self):
        from models.ninja import Ninja
        return Ninja
